doc/source/user-storage.md

```diff
@@ -0,0 +1,207 @@
+```eval_rst
```

Comment:
Love this!

Reply:
yeah I am warming to markdown+sphinx after figuring out their particular syntax for stuff :-)